docs/concepts/Glossary.md

```diff
@@ -0,0 +1,108 @@
+# Glossary
+
+#### _Aggregated Blueprint_
+
+  Aggregated Blueprints are Blueprints that bundle multiple other blueprints.
+  They contain intruction how they these referenced blueprints interact with each other.
+
+  Their pratical use is to install mutliple components that depend on each other.
+
+#### _Blueprint_
+
+  Blueprints contain actual intructions and steps on how to install a software component and what is needed to perform these actions.
+
+  __Background Knowledge__:
+    <details>
+    Blueprints consists of:
+      - Configuration Data ([Imports](#_import_))
+      - Installation intructions
+        - [DeployItems](#_deploy-items_) or
+        - [Subinstallations](#_subinstallations_)
+      - [Output](#_export_)
+    </details>
+
+#### _Context_
+
+  A context defines the scope in which an Installation runs and all of its data lives.
+  For every Installation a dedicated context is created and data can only be accessed within the same context.
+  Data can be exchanged between contexts via Import and Export declarations.
+
+  Since Installations can be nested, the resulting Contexts are nested as well.
+
+  For more information see [here](./Context.md).
+
+#### _DataObject_
+
+  DataObjects are vehicles to store arbitrary kinds of data. They exist in a [Context](#_context_) and provide data to Imports / receive data from Exports. They can be considerd to be the implementation of the data flow in an installation. 
+
+#### _Deployer_
+
+  Deployer are highly specialized controllers that act on [DeployItems](#_deployitem_) of a certain type. They execute the installation instructions and aim to maintain the declared desired state.
+
+#### _DeployExecutions_
+
+  A list of templates to generate [deploy items](#_deployitem_) as part of a [Blueprint](#_blueprint_). Mainly used to describe the installation instructions and customize it using the data provided as [imports](#_import_).
+
+#### _DeployItem_
+
+  A DeployItem is the interface between the Landscaper controller and the [Deployers](#_deployer_). It contains input data and a set of Deployer-specific instructions on how to install a component (e.g. install a helm chart with some custom values). Additionally, it is used to record the status as returned by the Deployer.
+
+#### _Execution_
```

Comment:
It seems that something is missing here ...

Reply:
the glossary ist just not yet completed.
But we rather wanted to push the changes in and continuously improve missing pieces.

Reply:
Ah, ok, then lgtm